doofinder.php

```diff
@@ -1250,6 +1255,28 @@ private function deleteItemsApi($hashid, $type, $payload)
         }
     }
 
+    private function indexApiInvokeReindexing()
+    {
+        require_once dirname(__FILE__) . '/lib/doofinder_api_index.php';
+
+        $region = Configuration::get('DF_REGION');
+        $api_key = $region . '-' . Configuration::get('DF_AI_APIKEY');
```

Comment:
```suggestion
        $api_key = Configuration::get('DF_API_KEY');
```
Use DF_API_KEY.
DF_AI_APIKEY is for autoinstaller only

Reply:
Changed on https://github.com/doofinder/doofinder-prestashop/pull/143/commits/f0c445600bbddbc8f382b7306a50c454ad70070a